stl/inc/yvals_core.h

```diff
@@ -974,6 +978,7 @@
 #define __cpp_lib_generic_unordered_lookup 201811L
 #define __cpp_lib_list_remove_return_type  201806L
 #define __cpp_lib_math_constants           201907L
+#define __cpp_lib_span                     201907L

```

Comment:
https://cplusplus.github.io/LWG/issue3274 specifies `201902L`.

Reply:
The issue speaks of the two earlier (major) papers that affect `span`. Given that 2 additional papers were acepted in Belfast that considerably change `span` (P1394R4 and P1872R0) should this be updated to a new value `201911L`?

Reply:
There are two questions: what should the Working Paper say, and what should we implement? For the Working Paper, you have an excellent point (and perhaps we should file an LWG issue). For the implementation, we need to implement whatever appears in the next Working Paper, even if we believe the value of the macro should be higher.

Reply:
You are absolutely correct, that one must implement whats in the working paper. I adopted the change. If the working draft changes this can easily be adopted anyway